tests/modules/meryl/count/main.nf

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { MERYL_COUNT } from '../../../../modules/meryl/count/main.nf'
+
```

Comment:
Maybe add a test here for multiple reads too? Might help identify any weirdities with PE data.